src/export/bops/index.ts

```diff
@@ -48,45 +49,43 @@ const bopsDictionary = {
 function isTypeForBopsPayload(type?: ComponentType) {
   if (!type) return false;
 
-  //To-Do Set MapAndLabel to True once we know shape of BOPs data
-
   switch (type) {
+    case ComponentType.Answer:
     case ComponentType.Calculate:
     case ComponentType.Confirmation:
     case ComponentType.Content:
     case ComponentType.DrawBoundary:
     case ComponentType.ExternalPortal:
+    case ComponentType.Feedback: // TODO
     case ComponentType.FileUpload:
     case ComponentType.FileUploadAndLabel:
     case ComponentType.Filter:
     case ComponentType.FindProperty:
     case ComponentType.Flow:
     case ComponentType.InternalPortal:
+    case ComponentType.MapAndLabel: // TODO - Parse GeoJSON FeatureCollection
```

Comment:
I was also going to pick this up here, but the data structure it outputs is actually different. This will be a follow up PR shortly!

Reply:
Please see https://github.com/theopensystemslab/planx-core/pull/562